packages/core-react/index.d.ts

```diff
@@ -8,4 +8,21 @@ export type TextFieldProps = {
 
 declare const TextField: React.FC<TextFieldProps>;
 
-export { TextField };
+export type ContainerProps = {
+  Tag?: React.ElementType<any>;
+} & React.HTMLProps<HTMLElement>;
+
+declare const Container: React.FC<ContainerProps>;
+
+export type GridProps = React.HTMLProps<HTMLElement>;
+
+declare const Grid: React.FC<GridProps>;
+
+export type CellProps = {
+  push?: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+  span?: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+} & React.HTMLProps<HTMLElement>;;
+
+declare const Cell: React.FC<CellProps>;
+
+export { Container, Grid, TextField };
```

Comment:
And `Cell` haha

Reply:
this was intentional: https://github.com/wework/ray/pull/213#discussion_r319401633 I am ok with exporting it though

Reply:
Ahh ok cool!